src/viur/shop/skeletons/order.py

```diff
@@ -9,9 +9,14 @@
 logger = SHOP_LOGGER.getChild(__name__)
 
 
-def get_payment_providers() -> dict[str, str | translate]:
+def get_payment_providers() -> dict[str, dict[str, t.Any]]:
```

Comment:
This method is used as SelectBone values callable and for the API. Which worked so far..
But now both points needs different outputs.
The bone values should stay a string:
![image](https://github.com/user-attachments/assets/61c37129-065c-4713-a456-4303a5f915e4)

Reply:
Should we split this in two functions like 
```python
def get_payment_providers() -> dict[str, str | translate]:
    return {
        pp.name: pp.title for pp in SHOP_INSTANCE.get().payment_providers
    }


def get_payment_providers_list() -> dict[str, dict[str, t.Any]]:
    return {
        pp.name: {
            "title": pp.title,
            "descr": pp.description,
            "image_path": pp.image_path,
        }

        for pp in SHOP_INSTANCE.get().payment_providers
    }

```

Reply:
Since this is used in the module, we could move this to the module. But we can refactor this later.
For now this seems to work fine.